packages/web-twig/src/Resources/components/Toast/README.md

```diff
@@ -0,0 +1,284 @@
+
+#### Toast Queue Limitations
+
+While the Toast queue becomes scrollable when it does not fit the screen, we recommend displaying only a few toasts at
+once for several reasons:
+
+⚠️ **We strongly discourage from displaying too many toasts at once as it may cause the page to be unusable,
+especially on mobile screens. As of now, there is no automatic stacking of the toast queue items. It is the
+responsibility of the developer to ensure that the Toast queue does not overflow the screen.**
+
+⚠️ Please note that scrolling is only available on pointer-equipped devices (mouse, trackpad). Furthermore, scrolling is
+only possible when the cursor is placed over the toast message boxes. This way the page content behind the toast
+messages can remain accessible.
+
+👉 Please note that the initial scroll position is always at the **top** of the queue.
+
+### API
+
+| Name         | Type                                                        | Default  | Required | Description                             |
+| ------------ | ----------------------------------------------------------- | -------- | -------- | --------------------------------------- |
+| `alignmentX` | [[AlignmentX dictionary][dictionary-alignment] \| `object`] | `center` | ✕        | Horizontal alignment of the toast queue |
+| `alignmentY` | [[AlignmentY dictionary][dictionary-alignment] \| `object`] | `bottom` | ✕        | Vertical alignment of the toast queue   |
+
+On top of the API options, the components accept [additional attributes][readme-additional-attributes].
+If you need more control over the styling of a component, you can use [style props][readme-style-props]
+and [escape hatches][readme-escape-hatches].
+
+## ToastBar
+
+The ToastBar component is the actual toast notification. It is a simple container with a message and a few optional
+elements.
+
+Minimum example:
+
+```twig
+<ToastBar>
+  Message only
+</ToastBar>
+```
+
+### Optional Icon
+
+An icon can be displayed in the ToastBar component, depending on the color of the ToastBar:
+
+```twig
+<ToastBar color="success" hasIcon>
+  Message with icon
+</ToastBar>
+```
+
+Alternatively, a custom icon can be used:
+
+```twig
+<ToastBar iconName="download">
+  Message with custom icon
+</ToastBar>
+```
+
+#### Default Icons according to Color Variant
+
+| Color name    | Icon name     |
+| ------------- | ------------- |
+| `danger`      | `danger`      |
+| `default`     | `info`        |
+| `informative` | `info`        |
+| `success`     | `check-plain` |
+| `warning`     | `warning`     |
+
+### Action Link
+
+An action link can be added to the ToastBar component:
+
+```twig
+<ToastBar>
+  Message with action
+  <Link href="#" color="inverted" isUnderlined>Action</Link>
+</ToastBar>
+```
+
+👉 For the sake of flexibility, developers can pass the link as part of the message. However, it is strongly recommended
+to use the **inverted underlined** variant of the link to make it stand out from the message.
```

Comment:
maybe we should mention that this doesn't change when you use colors. Eg. here you have inverted link in inverted toastbar. But you still use inverted link in success toastbar.